docs/server/source/server-reference/bigchaindb-cli.md

```diff
@@ -80,22 +80,27 @@ configuration file as documented under
 **This is an experimental feature. Users are advised not to use it in production.**
```

Comment:
Can we remove this warning line now, or should we wait until later? If we should wait until later, maybe put it as another item near the end of the list in issue #2372

Reply:
I think we should wait, so I updated the issue description here #2372

Reply:
We should certainly leave it until we have BEP-21 fully implemented. We can consider removing it when we have fully tested the new upsert-validator election process.